src/pages/Profile.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useEffect, useState } from "react";
+
+import UserDatails from "../components/UserDetails/UserDatails";
+
+const Profile = () => {
+  return (
+    <>
+      <UserDatails />
```

Comment:
if there will be no more components in this file IMO we should implement `UserDatails` here

Reply:
We need more components here, like the edit and add button, the blog form, the update form, and the user details update form.